legion/engine/physics/HalfEdgeFace.cpp

```diff
@@ -242,6 +244,18 @@ namespace legion::physics
         return face;
     }
 
+    void HalfEdgeFace::DEBUG_DrawFace(const math::mat4& transform,const math::color& debugColor,float time)
```

Comment:
is the capitals DEBUG needed? Also it could be interesting to have a debug::drawFace(HalfEdgeFace& face). which also draws the face